src/browser/navigation_validation.ts

```diff
@@ -52,8 +52,8 @@ export function navigationValidator(uuid: string, name: string, id: number) {
         const appObject = coreState.getAppObjByUuid(uuid);
         const appMetaInfo = coreState.appByUuid(uuid);
         const isMailTo = /^mailto:/i.test(url);
-        const allowed = isMailTo || validateNavigationRules(uuid, url, appMetaInfo.parentUuid, appObject._options) &&
-                                    isURLAllowed(url);
+        const allowed = isMailTo || (appObject && validateNavigationRules(uuid, url, appMetaInfo.parentUuid, appObject._options) &&
```

Comment:
Do we have to check `appMetaInfo` as well?

Reply:
I only added this check for typescript reasons, we don't currently test it and it builds.